src/feed_handler.py

```diff
@@ -388,8 +388,8 @@ def compare(self, other: StandardFeedHandler) -> bool:
         for cpid in self_books.keys():
             self_book = self_books[cpid]
             other_book = other_books[cpid]
-            comparison_failed = self_book.compare_books(other_book)
-            if comparison_failed:
+            book_match = self_book.compare_books(other_book)
```

Comment:
Use of variable was flipped; `compare_books` returns `True` if the book is the same; however this line previously assumed it returns `True` if mismatch